phpstan.neon

```diff
@@ -1,3 +1,5 @@
 parameters:
     ignoreErrors:
-        - '#Undefined variable: \$argv#'
+        - '#Variable \$reject might not be defined.#'
```

Comment:
As far as `phpstan/phpstan` is concerned, it's right: the variable might not be defined.

Other than that, I think it is safe to assume that it is defined, see https://github.com/phpbench/phpbench/blob/f72b3446c351d8f7ac82a41b30892739890b4d72/lib/Benchmark/Runner.php#L277-L280

Reply:
yeah, it will only enter that branch if there is at least one reject.